src/octod/media.d

```diff
@@ -26,6 +26,7 @@ module octod.media;
 enum ProtocolVersion
 {
     V3 = "v3",
+    JeanGreyPreview = "jean-grey-preview",
```

Comment:
Is that how they call v4?

Reply:
No, this is some itermediate version to get access to v4 global node_id's while still using v3

Reply:
https://developer.github.com/v3/#graphql-global-relay-ids